numpy/_core/defchararray.pyi

```diff
@@ -27,7 +28,7 @@ from numpy._typing import (
     _ArrayLikeStr_co as U_co,
     _ArrayLikeBytes_co as S_co,
     _ArrayLikeInt_co as i_co,
-    _ArrayLikeBool_co as b_co,
+    _ArrayLikebool_co as b_co,
```

Comment:
This change looks incorrect, it's still `_ArrayLikeBool_co` in `numpy._typing`.

Reply:
Right - fixed it.